src/Sarif.Viewer.VisualStudio/ErrorList/SarifResultTableEntry.cs

```diff
@@ -182,19 +182,15 @@ private void ErrorListInlineLink_Click(object sender, RoutedEventArgs e)
 
                 if (location != null)
                 {
-                    // Set the current SARIF error in the manager so we track code locations.
-                    CodeAnalysisResultManager.Instance.CurrentSarifResult = this.Error;
-
+                    // Setting the DataContext to null forces the TabControl to select the appropriate tab.
                     SarifViewerPackage.SarifToolWindow.Control.DataContext = null;
 
                     if (this.Error.HasDetails)
                     {
-                        // Setting the DataContext to null (above) forces the TabControl to select the appropriate tab.
                         SarifViewerPackage.SarifToolWindow.Control.DataContext = this.Error;
```

Comment:
>   SarifViewerPackage.SarifToolWindow.Control.DataContext = this.Error; [](start = 21, length = 71)

This should go away. There should be a a way to set the "navigated" item on the selection service. This code should not be reaching into the explorer window.